tools/cli/commands/watch.js

```diff
@@ -85,7 +86,7 @@ export async function watchCli( options ) {
  * @param {object} packageJson - The project's package.json file, parsed.
  */
 export async function watch( project, packageJson ) {
-	const command = hasWatchStep( project, packageJson );
+	const command = checkPropjectScriptAvailability( project, 'watch', packageJson );
```

Comment:
Not due to this PR, but I noticed the doc comment is wrong when flagging the typo on this line.
```suggestion
 * @param {object} composerJson - The project's composer.json file, parsed.
 */
export async function watch( project, composerJson ) {
	const command = checkProjectScriptAvailability( project, 'watch', composerJson );
```